features/site-overall.feature

```diff
@@ -0,0 +1,14 @@
+Feature: A user visit gigadb website
+  As a gigadb user
+  All front end views display properly
+  So that I can experience a good performance of gigadb website
+
+  Background:
+    Given Gigadb web site is loaded with production-like data
+
+  @ok @issue-883
```

Comment:
I don't think we need the `@issue-883` tag. It's not a tag that I would expect to use in the future to execute a set of acceptance tests.